tests/test_common_gpu.py

```diff
@@ -1083,6 +1084,50 @@ def test_8bit_dora_merging(self):
         assert torch.allclose(out_dora, out_unmerged, atol=atol, rtol=rtol)
         assert torch.allclose(out_dora, out_unloaded, atol=atol, rtol=rtol)
 
+    @require_torch_gpu
+    @pytest.mark.single_gpu_tests
+    @require_bitsandbytes
+    def test_8bit_dora_cpu_offloading(self):
+        torch.manual_seed(0)
+        model = AutoModelForCausalLM.from_pretrained(
+            "facebook/opt-125m",
+            quantization_config=BitsAndBytesConfig(load_in_8bit=True),
+            torch_dtype=torch.float32,
+        ).eval()
+
+        config = LoraConfig(
+            r=4096, # too small and the time difference is too small
+            init_lora_weights=False,
+            use_dora=True,
+        )
+        peft_model = get_peft_model(model, config).eval()
+
+        # Save to disk
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            peft_model.save_pretrained(tmp_dir)
+
+            # Load from disk 100% on CPU without ephemeral transfers
+            start_time = time.time()
```

Comment:
```suggestion
            start_time = time.perf_counter()
```

Let's use `perf_counter` here and below.